spec/controllers/projects_controller_spec.rb

```diff
@@ -1,13 +1,85 @@
 require 'spec_helper'
 
+RSpec.shared_examples 'denies access' do
+  before { request }
+
+  it 'redirects to the root path' do
+    expect(response).to redirect_to(root_path)
```

Comment:
Yeah thats the way its handled, I think we can move this test to a context in the other tests so we can test the behaviour when it happens there.